src/components/Photo/PhotoDragDrop.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useRef, useState } from 'react';
+
+import { Box, Typography, useTheme } from '@mui/material';
+import { Button, Icon } from '@terraware/web-components';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+
+import strings from 'src/strings';
+
+export type PhotoDragDropProps = {
+  files: File[];
+  setFiles: (files: File[]) => void;
+  uploadText?: string;
+  uploadDescription?: string;
+  uploadMobileDescription?: string;
+  photoSelectedText?: string;
+  chooseFileText?: string;
+  replaceFileText?: string;
+  maxPhotos?: number;
+  multipleSelection?: boolean;
+};
+
+const PhotoDragDrop = (props: PhotoDragDropProps) => {
+  const {
+    multipleSelection,
+    uploadText = strings.UPLOAD_PHOTOS,
+    uploadMobileDescription = strings.UPLOAD_PHOTO_DESCRIPTION,
+    uploadDescription,
+    photoSelectedText,
+    chooseFileText = strings.CHOOSE_FILE,
+    replaceFileText = strings.REPLACE_FILE,
+    maxPhotos,
+    files,
+    setFiles,
+  } = props;
+  const { isMobile } = useDeviceInfo();
+  const theme = useTheme();
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [editing, setEditing] = useState<boolean>(false);
+
+  const addFiles = (fileList: FileList) => {
+    const newFiles: File[] = [];
+
+    for (let i = 0; i < fileList.length; i++) {
+      const fileItem = fileList.item(i);
+      if (fileItem) {
+        newFiles.push(fileItem);
+      }
+    }
+
+    if (newFiles.length) {
+      setFiles([...files, ...newFiles].slice(0, maxPhotos));
+    }
+  };
+
+  const dropHandler = (event: React.DragEvent<HTMLDivElement>) => {
+    event.preventDefault();
+    addFiles(event.dataTransfer.files);
+  };
+
+  const onFileChosen = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (editing) {
+      setEditing(false);
+    }
+    if (event.currentTarget.files) {
+      addFiles(event.currentTarget.files);
+    }
+  };
+
+  const onChooseFileHandler = () => {
+    inputRef.current?.click();
+    document.querySelectorAll<HTMLElement>('.photo-button').forEach((el) => el.blur());
+  };
```

Comment:
I couldn't think of a better way to do this, so open to suggestions. Previously we were calling `divRef.current?.focus();` which was just some random other element, that unfortunately doesn't exist in this smaller component. I tried with several other components in this one and couldn't get it to work (including adding a `ref={ref}` to the Button element in web-components).

Reply:
I don't remember why we needed that blur

Reply:
I'm guessing it's just because the button has a focus border around it after selecting, and it shouldn't really anymore?